pom.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2016 Johns Hopkins University
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4.0.0.xsd">
+         
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.dataconservancy.packaging.shared</groupId>
+    <artifactId>dcs-packaging-shared</artifactId>
+    <packaging>jar</packaging>
+    <name>DC Packaging Shared</name>
+    <description>Shared code for Data Conservancy Packaging</description>
+    <version>1.0.3</version>
```

Comment:
What version should be set for this new package?  Does it start at 1.0.0 or 1.0.1, or should it match its companion projects, which seem to be mainly at 1.0.3?